packages/xo-web/src/xo-app/xoa/update/index.js

```diff
@@ -178,6 +213,10 @@ const Updates = decorate([
         xoaTrialState.state === 'default' &&
         !isTrialRunning(xoaTrialState.trial) &&
         !exposeTrial(xoaTrialState.trial),
+      consolidatedChannel: ({ channel }, { xoaConfiguration }) =>
```

Comment:
Keep the computed sorted please :slightly_smiling_face:

Reply:
:heavy_check_mark: